pkgs/applications/networking/cluster/docker-machine/xhyve.nix

```diff
@@ -1,24 +1,34 @@
-{ stdenv, buildGoPackage, fetchFromGitHub, pkgconfig, Hypervisor, vmnet }:
+{ stdenv, buildGoPackage, fetchFromGitHub, pkgconfig, cctools, Hypervisor, vmnet }:
 
 buildGoPackage rec {
   pname = "docker-machine-xhyve";
-  version = "0.3.3";
+  version = "0.4.0";
 
   goPackagePath = "github.com/zchee/docker-machine-driver-xhyve";
   goDeps = ./xhyve-deps.nix;
 
+  # https://github.com/machine-drivers/docker-machine-driver-xhyve/pull/225
+  patches = [ ./xhyve-lib9p-EPERM-fix.patch ];
```

Comment:
please use `fetchpatch` with the single commit https://github.com/machine-drivers/docker-machine-driver-xhyve/commit/546256494bf2ccc33e4125bf45f504b0e3027d5a.patch